src/middlewared/middlewared/plugins/jail.py

```diff
@@ -224,29 +224,44 @@ def stop(self, jail):
         Str("jail"),
         Dict(
             "options",
-            Str("action"),
+            Str("action", enum=["ADD", "EDIT", "REMOVE", "REPLACE", "LIST"]),
             Str("source"),
             Str("destination"),
             Str("fstype"),
             Str("fsoptions"),
             Str("dump"),
-            Str("_pass"), ))
+            Str("_pass"),
```

Comment:
Why `_pass` with underscore? Python reserved word? Any reason it cant just be `pass` in the dict and use `_pass` within the python method?

Reply:
Reason I ask is because javascript client might wonder why such a weird name =P

Reply:
Yep reserved word ;)

In addition it's the same way I dodge the usage in iocage so I kept it for consistency sake. It could certainly be changed ;) 

The only argument against is Lola is currently using this and that change is just for visual sake. No solid reason to do so imo and it helps silence my editor :D

Reply:
There is no reason it could not be `pass` just for the dict, but fair enough